chart/templates/rbac/pod-launcher-rolebinding.yaml

```diff
@@ -21,7 +21,7 @@
 {{- if and .Values.rbacEnabled .Values.allowPodLaunching }}
 {{- $grantScheduler := or (eq .Values.executor "LocalExecutor") (eq .Values.executor "SequentialExecutor") (eq .Values.executor "KubernetesExecutor") }}
 {{- $grantWorker := or (eq .Values.executor "CeleryExecutor") (eq .Values.executor "KubernetesExecutor") }}
-kind: RoleBinding
+kind: ClusterRoleBinding
```

Comment:
why do we need `ClusterRoleBinding` ? we would usually only need these Role and RoleBinding  to apply to the namespace where airflow is bing installed. `ClusterRoleBinding` will create issues in managed multitenant kubernetes  clusters which only provide namespace level access(and not cluster wide)

Is there a test which doesn't work with `Role` and `RoleBinding`

Reply:
I think this is a good question - to @dimberman -> was it needed or was it just a remnant of some experimentation?

Reply:
The astronomer dropped support for deployment in one namespace. 
You can read more about this feature in this PR.
https://github.com/astronomer/docs/pull/216/files

Reply:
Hey guys the original PR of the chart does have `RoleBinding` since this role is attached to the scheduler and workers so that they can launch `KubernetesExecutor` pods and `KubernetesPodOperator` pods.

In the Astronomer world, these components only launch pods into their own namespace and not across other namespaces, which is why this was originally a `RoleBinding`.

This PR seems to be changing that `RoleBinding` to a `ClusterRoleBinding`. I assume its so that we can launch pods into a different namespace than where the scheduler (and/or workers) are running. @potiuk is that what you ran into here?

Reply:
Correct @schnie . I have not written the tests myself but we have indeed "airflow" runnig in airflow namespace but all the pods run by it are executed in the "default" namespace.

Reply:
@potiuk really? When I ran it locally it seemed like all the k8sexecutor pods were run in the airflow namespace, and it was just the k8soperator pods in default (we can change those to airflow if we want)

Reply:
That said, I think we can still merge this and decide this after. I want to get this into v1-10-test ASAP so I can start running integration tests.